resources/windows/msi/service.mcollective.wxs

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <DirectoryRef Id="TBD">  # The ID has to match the Ruby Bin directory
```

Comment:
As per discussion we had on hipchat I will document it here. This could likely be the autogenerated output of the ID field of the wix directory structure from https://github.com/puppetlabs/vanagon/pull/283 if we choose to go down that route and refactor that PR. Or alternatively it could be one of the "wix_id" params defined in that same PR